Formula/qt.rb

```diff
@@ -91,6 +88,35 @@ def install
     # It is not friendly to Homebrew or macOS
     # because on macOS `/tmp` -> `/private/tmp`
     inreplace "qtbase/CMakeLists.txt", "FATAL_ERROR", ""
+    # WORKAROUND: build gn first
+    if OS.mac?
```

Comment:
Why do we need the third and fourth `inreplace`? Is there an upstream bug report?

Reply:
In fact `qt` doesn't support build path containing symlink https://github.com/qt/qtbase/blob/a8823db57839f0a690f1cd488e2a268871aeca68/CMakeLists.txt#L21. It causes `gn` create some headers in an unexpected folder.